common/src/main/java/org/apache/celeborn/common/meta/FileInfo.java

```diff
@@ -56,7 +56,7 @@ public long getFileLength() {
     return bytesFlushed;
   }
 
-  public void updateBytesFlushed(long bytes) {
+  public synchronized void updateBytesFlushed(long bytes) {
     bytesFlushed += bytes;
```

Comment:
I am not sure about this change - it is not clear to me why we are locking on `this` here.
I will revisit this later if you believe this is the right change - not sure if subclasses of this abstract class need this ... if true, n general I would expect those kind of patterns to be difficult to evolve.

Reply:
@mridulm, you could take a look at the description of this pull request, which has listed all WARNINGs.

Reply:
Scratch that - this change makes sense: not sure what I missed when I reviewed it first time around :-(